tekton/pipeline_run.go

```diff
@@ -48,6 +50,9 @@ const (
 
 	//PipelineTypeRelease is the type for PipelineRuns created to run a release Pipeline
 	PipelineTypeRelease = "release"
+
+	// pipelinesAsCodeLabelsPrefix is the prefix for pipelines-as-code metadata
```

Comment:
Seems it should say `pipelinesAsCodeMetadataPrefix` not `pipelinesAsCodeLabelsPrefix`